src/cli.cpp

```diff
@@ -203,7 +216,7 @@ int CLI::parseCommandLine(int argc, char* argv[], Options* options) {
     }
 
     // make sure that a product id is entered for OFFICE_2K3 or OFFICE_2K7 IIDs
-    if ((options->activationMode == OFFICE_2K3 || options->activationMode == OFFICE_2K7) && options->productid == "") {
+    if ((options->activationMode == OFFICE_2K3 || options->activationMode == OFFICE_2K7) && (options->productid.empty() || options->instid.empty()) ) {
```

Comment:
`inst id` is required for my office 2007 enterprise activation to generate a numeric value, otherwise an invalid alphanumeric (dialog accepts only numeric) is returned

i dont have std office 2007/2003 to test - if alphanumeric value accepted by those programs we will have to split out enterprise /non-enterprise

Reply:
> `inst id` is required for my office 2007 enterprise activation to generate a numeric value, otherwise an invalid alphanumeric (dialog accepts only numeric) is returned
> 
> 
> 
> i dont have std office 2007/2003 to test - if alphanumeric value accepted by those programs we will have to split out enterprise /non-enterprise

the program returned a Windows XP VLK "product key" instead of a "Confirmation ID" as the default operation of the program if no cmd options are given is to generate a Wi dows XP VLK product key. since you didnt specify "-i" in cmd line the program defaulted to product key generation mode.